nano/node/rpc.cpp

```diff
@@ -708,7 +708,8 @@ void nano::rpc_handler::account_representative_set ()
 			{
 				bool generate_work (work == 0); // Disable work generation if "work" option is provided
 				auto response_a (response);
-				wallet->change_async (account, representative, [response_a](std::shared_ptr<nano::block> block) {
+				// clang-format off
+                wallet->change_async (account, representative, [response_a](std::shared_ptr<nano::block> block) {
```

Comment:
Something happened to indentation here it seems, and with clang-format off there're no complaints :D

Reply:
What should I do here? :) This is what happens when I tab it to the correct place and run clang-format

Reply:
Using my mac for checking in for the first time, my Xcode preferences were changing tabs to spaces. Fixed :)